localisation/00_wc_sad.csv

```diff
@@ -517,11 +517,11 @@ EVTDESC_WCOME.34;You can feel the power of Yogg-Saron all over your body. The gr
 EVTOPTA_WCOME.34;I will bring ruin to enemies of Yogg-Saron.;;Ich werde den Feinden von Yogg-Saron Tod und Ruin bringen.;;;;;;;;;;;x
 EVTDESC_WCOME.35;You can feel the power of Yogg-Saron all over your body. The great eye overtakes your mind and enters the depths of your consciousness. Though the Master is pleased with you, his gratitude is a little more than you can handle.;;Ihr k�nnt die Kraft von Yogg-Saron am ganzen Leib sp�ren. Das gro�e Auge �berw�ltigt Euren Geist und dringt in die Tiefen Eures Bewusstseins vor. Obwohl der Meister mit Euch zufrieden ist, ist seine Dankbarkeit ein wenig mehr, als Ihr ertragen k�nnt.;;;;;;;;;;;x
 EVTOPTA_WCOME.35;UULL lwhuk H'IWN!;;UULL lwhuk H'IWN!;;;;;;;;;;;x
-EVTDESC_WCOME.200;You have entered the undergrounds of Ahn'Qiraj with a small band of elite adventurers. After making it through horrific forces of C'Thun you now stand before him. The maddening whispers enter your mind as soon as you look at him. You overcome the madness and lead the charge against the Old God.;;Ihr habt die Unterwelt von Ahn'Qiraj mit einer kleinen Gruppe von Elitabenteurern betreten. Nachdem ihr es durch die schrecklichen Kr�fte von C'Thun geschafft habt, steht ihr nun vor ihm. Das wahnsinnige Fl�stern dringt in Euren Geist ein, sobald Ihr ihn anseht. Ihr �berwindet den Wahnsinn und f�hrt den Angriff auf den alten Gott an.;;;;;;;;;;;x
-EVTOPTA_WCOME.200;Die, you monster.;;Stirb, du Ungeheuer.;;;;;;;;;;;x
-EVTDESC_WCOME.201;A small band of elite adventurers led by [From.GetBestName] has vanquished the Old God. The band has overcome his minions and cleared out the city of Ahn'Qiraj. The races of Azeroth can feel safe once again.;;Eine kleine Gruppe von Eliteabenteurern unter der Leitung von [From.GetBestName] hat den alten Gott besiegt. Die Gruppe hat seine Schergen �berwunden und die Stadt Ahn'Qiraj befreit. Die V�lker Azeroths k�nnen sich wieder sicher f�hlen.;;;;;;;;;;;x
+EVTDESC_WCOME.200;You entered Ahn'Qiraj with a small band of soldiers. After overcoming every challenge thrown at you and your men, despite heavy losses, you finally stand before one of the mightiest enemies you will ever face! His whispers enter your mind as soon as your gaze upon the horrific creature. However, with your strong will you overcome the madness and lead the charge against C'Thun.;;Ihr habt Ahn'Qiraj mit einigen Soldaten betreten. Nachdem ihr jede Herausforderung gemeistert habt, steht ihr schlie�lich vor einem der m�chtigsten Feinde, denen Ihr je entgegentreten werdet! Als Ihr die schreckliche Kreatur erblickt, dringt sein Fl�stern in Euren Geist ein. Mit  starkem Willen �berwindet Ihr den Wahnsinn und f�hrt den Angriff auf C'Thun an.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCOME.200;You enter Ahn'Qiraj with a small band of soldiers. After overcoming every challenge thrown at you and your men despite heavy losses, you finally stand before one of the mightiest enemies that you will ever face! His whispers enter your mind as soon as your gaze upon the horrific visage. Nevertheless, you resist the call to madness and lead the charge against C'Thun with fervent will.;;Ihr habt Ahn'Qiraj mit einigen Soldaten betreten. Nachdem ihr jede Herausforderung gemeistert habt, steht ihr schlie�lich vor einem der m�chtigsten Feinde, denen Ihr je entgegentreten werdet! Als Ihr die schreckliche Kreatur erblickt, dringt sein Fl�stern in Euren Geist ein. Mit  starkem Willen �berwindet Ihr den Wahnsinn und f�hrt den Angriff auf C'Thun an.;;;;;;;;;;;x
```

Reply:
I thin, it's fine as it is. It tells, what you did before (enter the city, overcoming challenges) and when you stand before C'Thun, it switches to present. However, spotted a double-space in german, thx :)